src/app/models.py

```diff
@@ -123,6 +126,31 @@ def __init__(self, login, password, user_id, url, title, comment):
         self.comment = comment
 
 
+class FilterUserBy:
```

Comment:
May be better if class name will be FilterUser, and function names will be by_email(), by_username() and by_id().

Reply:
no, cause as I remember function name must be verbs